### DISCUSSION TITLE
License on civitai.com versus github repo

### DISCUSSION THREAD

**Discussion by akylaaa** [civitai/civitai#230] [Q&A]
I noticed in civitai.com license that UGC can only be used non-commercially by others, while here i read that commercial use is allowed. Are the models/embeddings on civitai.com different from the models/embeddings here, or am i missing something?

**Comment by WASasquatch** [comment]
There is a lot of licensing issues on civitai, such as allowing users to specify arbitrary stipulations like non-commercial, when the underlying model license everything they make with it supersedes such restrictions and renders them null-and-void, and those licenses must ship with the models, which civitai doesn't even support yet. :O
